### ISSUE TITLE
Its not loaded or hooked in time

### ISSUE THREAD

**Issue by mmuziek** [Rocologo/BagOfGold#19]
Hi there,
i use your plugin but sadly it hooks into vault way to late.
this causes all plugins  that load between vault and this plugin to be broken and crashing due to no economy found.
i never had this issue before.
im using BOG : 1.0.6
and Mobhunter : 5.5.6
And vault : Dev Build from MD_5 https://ci.md-5.net/job/Vault/
And protocol Lib : http://ci.dmulloy2.net/job/ProtocolLib/390/
it starts fine it seems 

[14:55:19] [Server thread/INFO]: [BagOfGold] Enabling BagOfGold v1.0.6
[14:55:19] [Server thread/INFO]: [BagOfGold] =====================WARNING=============================
[14:55:19] [Server thread/INFO]: The statistics collection is disabled. As developer I need the
[14:55:19] [Server thread/INFO]: statistics from bStats.org. The statistics is 100% anonymous.
[14:55:19] [Server thread/INFO]: https://bstats.org/plugin/bukkit/bagofgold
[14:55:19] [Server thread/INFO]: Please enable this in /plugins/bStats/config.yml and get rid of this
[14:55:19] [Server thread/INFO]: message. Loading will continue in 15 sec.
[14:55:19] [Server thread/INFO]: [BagOfGold] =========================================================
[14:55:34] [Server thread/INFO]: [BagOfGold] Database version 1 detected.
[14:55:34] [Server thread/INFO]: [BagOfGold] Enabling compatibility with Citizens2 (2.0.22-SNAPSHOT (build 1479))
[14:55:34] [Server thread/INFO]: [BagOfGold][Economy] BagOfGold hooked into Vault
[14:55:34] [Server thread/INFO]: [BagOfGold] [BagOfGold][Economy] BagOfGold found: Loaded

i do wanna state i tried the old one and MD-5 his vault so its not vault

**Comment by Rocologo** [comment]
I need the plugin names from the server log, i think i can ask bog to load before these plugins in the plugin.yml file

**Comment by mmuziek** [comment]
https://pastebin.com/sZu1pw6q
there ya go

**Comment by Rocologo** [comment]
Please test build 60
https://fractal.lindegaard.one:8181/job/BagOfGold/60/

It should force BagOfGold to load before : ChestCommands, MCGMineStore, BitcubesCore, BetonQuest, Stickers

**Comment by Rocologo** [comment]
Please let me know if this solves the problem.

**Comment by mmuziek** [comment]
sadly the following plugins still give a no economy error and it still starts after those plugins : 
BetonQuest
AcidIsland
BetterShops
SuperLuckyBlock-pro
CMI
ChestCommands
TradeMe
MobArena
Dragontravel
ect.....
i think we have a problem : 
https://pastebin.com/C40TCDDd
almost all plugins load before this one xD we can keep going this....

**Comment by Rocologo** [comment]
I downloaded BetonQuest on my TestServer and build 63 seems to work now.

http://jenkins.lindegaard.one:8180/job/BagOfGold/63/

BetomQuest hooks into BagOfGold on my test server now.

**Comment by Rocologo** [comment]
You can join me on Discord if you like https://discord.gg/JeMpMKT

**Comment by Rocologo** [comment]
I have found the problem :-) 

```
[Vault] Enabling Vault v1.5.6-b49
[22:36:22 INFO]: [Vault] [Permission] PermissionsEx found: Waiting
[22:36:22 INFO]: [Vault] [Permission] SuperPermissions loaded as backup permission system.
[22:36:22 INFO]: [Vault] [Chat] PermissionsEx found: Waiting
[22:36:22 INFO]: [Vault] Enabled Version 1.5.6-b49
[22:36:22 INFO]: [BagOfGold] Enabling BagOfGold v1.0.7-SNAPSHOT-BTestBuild
[22:36:23 INFO]: [MobHunting] Config.yml was backed up to plugins/MobHunting/backup/config.yml.bak52
[22:36:23 INFO]: [BagOfGold] Config.yml was backed up to plugins/BagOfGold/backup/config.yml.bak358
[22:36:23 INFO]: [BagOfGold] Database version 1 detected.
[22:36:23 INFO]: [BagOfGold][Economy] BagOfGold hooked into Vault
[22:36:23 INFO]: [BagOfGold] [BagOfGold][Economy] BagOfGold found: Loaded
[22:36:23 INFO]: [BetonQuest] Enabling BetonQuest v1.9.6
[22:36:23 INFO]: [BetonQuest] Using SQLite for storing data!
[22:36:23 INFO]: [BetonQuest] Configuration up to date!
[22:36:23 INFO]: [BetonQuest] Hooking into Vault
[22:36:23 INFO]: [BetonQuest] Autoupdater enabled!
[22:36:23 INFO]: [BetonQuest] BetonQuest succesfully enabled!

```
Its actually a bug in BetonQuest!!! 

You can fix it yourself until the DEV fixes his plugin.yml file. I will report the problem to the DEV.

But If you want to fix it you can open the BetonQuest.jar with Your Archive Manager and edit the plugin.yml file yourself.

In the plugin.yml file the DEV has made this action: 
```
loadbefore: 
- Skript
- Citizens
- MythicMobs
- Multiverse-Core
- EffectLib
- PlayerPoints
- Heroes
- Magic
- Denizen
- WorldEdit
- WorldGuard
- mcMMO
- Shopkeepers
- SkillAPI
- Quests
- PlaceholderAPI
- HolographicDisplays
- RacesAndClasses
- LegendQuest
```
If you delete the line 
- Vault
and save 

Everything will work :-)

**Comment by mmuziek** [comment]
wel it doesnt work still :(
[10:53:02] [Server thread/INFO]: [BetonQuest] Hooking into Vault
[10:53:02] [Server thread/ERROR]: [BetonQuest] There is no economy plugin on the server!
https://pastebin.com/hQx1SMbp

**Comment by Rocologo** [comment]
DId you use the JAR I made or did you change plugin.yml yourself?

**Comment by mmuziek** [comment]
i tried both

**Comment by mmuziek** [comment]
here a new log
same problem
https://pastebin.com/WvNwTB98 
without beton installed

**Comment by Rocologo** [comment]
After talking to the DEV of BetonQuest I have done some changes in my own plugin (which I really dont like, but I have to try it) I have tested with BetonQuest and both with and without Essentials, and it worked on my test server.

Please test build 65 
https://fractal.lindegaard.one:8181/job/BagOfGold/

**Comment by Rocologo** [comment]
fixed in v1.0.7
